src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/Visit.kt

```diff
@@ -74,6 +74,15 @@ class Visit(
   @OneToMany(fetch = FetchType.LAZY, cascade = [CascadeType.ALL], mappedBy = "visit", orphanRemoval = true)
   var visitNotes: MutableList<VisitNote> = mutableListOf(),
 
+  @Column
+  var createdBy: String? = null,
```

Comment:
can this one be null?

Reply:
thanks, setting this as NOT NULL and as suggested added a default value of NOT_KNOWN